tests/ignite/engine/test_custom_events.py

```diff
@@ -531,4 +583,5 @@ def execute_some_handler(e):
 
     _test(Events.ITERATION_STARTED(once=1) | Events.ITERATION_STARTED(once=1), [1, 1])
     _test(Events.ITERATION_STARTED(once=1) | Events.ITERATION_STARTED(once=10), [1, 10])
-    _test(Events.ITERATION_STARTED(once=1) | Events.ITERATION_STARTED(every=3), [1, 3, 6, 9, 12, 15])
```

Comment:
Why have you deleted the statement testing `every`?

Reply:
Sorry, it's an accident, fixed.